apps/authentication-portal/src/main/webapp/basicauth.jsp

```diff
@@ -314,6 +314,7 @@
                     <%=AuthenticationEndpointUtil.i18n(resourceBundle, "continue")%>
             </button>
         </div>
+        <div></div>
```

Comment:
Shall we format the divs to increase readability? WDYT?

Reply:
This an unwanted line. Better remove it.

Reply:
This div tag is needed to clear the float of two elements above. Without this tag, this is how the buttons will be displayed:
![image](https://user-images.githubusercontent.com/20745428/73514437-c5b2fe00-4417-11ea-9ce8-8532bfc80946.png)

Reply:
You can clear float using css pseudo `:after` property.